pkg/synthesis/model/symbolicexpr/groupBasedInternalResource.go

```diff
@@ -49,3 +50,26 @@ func getConjunctionForGroups(isExclude bool, groups []*collector.Group, groupToC
 	}
 	return res
 }
+
+// return the []atomic corresponding to a given condition
+func getAtomicsForPath(containerModel *collector.ResourcesContainerModel,
+	isExcluded bool, pathExpr *collector.PathExpression, group string) []atomic {
+	res := []atomic{}
+	for _, path := range pathExpr.Paths {
+		groupOfPath := containerModel.FindGroupByPath(path)
+		segmentOfPath := containerModel.GetSegment(path)
+		switch {
+		case groupOfPath != nil:
+			res = append(res, groupAtomicTerm{group: groupOfPath, atomicTerm: atomicTerm{neg: isExcluded}})
+		case segmentOfPath != nil:
+			segment, err := configuration.ProcessSegment(segmentOfPath)
+			if err != nil {
+				debugMsg(group, err.Error())
+			}
+			res = append(res, SegmentTerm{segment: segment, atomicTerm: atomicTerm{neg: isExcluded}})
+		default:
+			debugMsg(group, fmt.Sprintf("includes a path %v which is not the current supported group or segment", path))
```

Comment:
```suggestion
			debugMsg(group, fmt.Sprintf("includes a path %s which is not the current supported group or segment", path))
```
please change prefix to `"group %s includes a path ... " `

Reply:
there was an additional comment here, apart from the suggested code change